packages/compass-crud/src/stores/crud-store.spec.ts

```diff
@@ -188,6 +188,16 @@ describe('store', function () {
           instance,
           preferences,
           logger: createNoopLoggerAndTelemetry(),
+          favoriteQueryStorageAccess: {
+            getStorage() {
+              return new CompassFavoriteQueryStorage();
+            },
+          },
+          recentQueryStorageAccess: {
+            getStorage() {
+              return new CompassRecentQueryStorage();
+            },
+          },
```

Comment:
Fwiw, if you want to add something like

```js
const favoriteQueryStorageAccess = {
  getStorage() {
    return new CompassFavoriteQueryStorage();
  },
};
const recentQueryStorageAccess = {
  getStorage() {
    return new CompassRecentQueryStorage();
  },
};
```

at he top of the file and then juts use

```js
          favoriteQueryStorageAccess,
          recentQueryStorageAccess,
```

here instead of providing a new definition each time, I think that would be okay and maybe reduce the verbosity level a bit 🙂 

Alternatively, maybe we could even just export `favoriteQueryStorageAccess` and `recentQueryStorageAccess` (with `compass...` name prefixed) from `'@mongodb-js/my-queries-storage'` as utilities so that consumers in test files don't need to repeat this bit at all?

Reply:
Ahh yea that makes sense, looks simple enough so I will get it in right away. Thank you :)